cli/internal/git/utils.go

```diff
@@ -63,10 +63,10 @@ func credentialParser() []Credential {
 	scanner := bufio.NewScanner(credentialsFile)
 	for scanner.Scan() {
 		gitUrl, err := url.Parse(scanner.Text())
-		password, _ := gitUrl.User.Password()
 		if err != nil {
```

Comment:
Because of the way we're using the scanner I understand that logging this error here probably doesn't make sense. 

@jeff-mccoy Would it be reasonable to check that there is >= 1 item in the credentials after we're done parsing through the scanner and log a warning or return an err (which the functions in this file are 'legacy Jeff' and don't really do)? 

If the latter, then that def shouldn't be in this PR. Just putting the question out there since it's getting touched here.

Reply:
Agreed, maybe as a debug, but an empty slice returned I think is still valid.  Also it is totally valid that we might no creds at all.